src/nvim/ex_cmds2.c

```diff
@@ -641,11 +645,11 @@ void ex_breaklist(exarg_T *eap)
  * Find a breakpoint for a function or sourced file.
  * Returns line number at which to break; zero when no matching breakpoint.
  */
-linenr_T 
-dbg_find_breakpoint (
-    int file,                   /* TRUE for a file, FALSE for a function */
-    char_u *fname,         /* file or function name */
-    linenr_T after             /* after this line number */
+linenr_T
+dbg_find_breakpoint(
+    int file,                   // TRUE for a file, FALSE for a function
+    char_u *fname,              // file or function name
+    linenr_T after              // after this line number
```

Comment:
I think changing signature doc is fine if you makes changes to signature anyways (or obviously if required by the linter as in this case), but why not change to the correct doxygen style directly? (if you are not familiar look i e in `charset.c` or `diff.c` which already are refactored)